Sources/VLCEditController.swift

```diff
@@ -82,7 +85,7 @@ private extension VLCEditController {
         alertController.addAction(cancelButton)
         alertController.addAction(confirmAction)
 
-        UIApplication.shared.keyWindow?.rootViewController?.present(alertController, animated: true, completion: nil)
+        present(alertController, animated: true, completion: nil)
```

Comment:
I think there are other usages of the `rootViewController` to present alerts.

Reply:
I know but it's only there because it's in code where you don't have access to the controller that wants to present it